src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx

```diff
@@ -6,7 +6,11 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-Some of our older customers are still on our **original pricing model**. Our newer pricing model, sometimes referred to as the **New Relic One pricing model**, is much simpler and offers many benefits compared to our original pricing model ([learn more](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing)). This doc explains the options available for switching to the newer pricing model. 
+<Callout variant="important">
+  This doc references our original product-based pricing model. For more about pricing changes, see [Overview of our pricing models](/docs/accounts/accounts-billing/new-relic-one-pricing-users/transition-guide-new-pricing-plan/).
+</Callout>
+
+Some of our older customers may not have had an opportunity to update their pricing model yet and are still on our **original pricing model**. Our newer pricing model, sometimes referred to as the **New Relic One pricing model**, is much simpler and offers many benefits compared to our original pricing model ([learn more](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing)). This doc explains the options available for switching to the newer pricing model. 
 
 ## How free accounts switch
```

Comment:
might need an anchor tag

Reply:
@akristen for the link in line 13, we want users to start reading at the top, so I don't think there's a need for an anchor. The same applies for the link in line 10.